__tests__/app.test.js

```diff
@@ -132,3 +133,41 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/article_id/comments", () => {
+  test("POST: 201 Adds a comment for an article", () => {
+    const userObj = { username: "butter_bridge", body: "This is a comment!" };
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(userObj)
+      .expect(201)
+      .then((response) => {
+        expect(typeof response.body.comment.comment_id).toBe("number");
+        expect(typeof response.body.comment.article_id).toBe("number");
```

Comment:
realistically, you can expect this to be the number 3, because that's on the path you've posted to :)